src/metabase/upload.clj

```diff
@@ -429,7 +429,7 @@
   (/ (.length ^File csv-file) 1048576.0))
 
 (defn- load-from-csv!
-  "Loads a table from a CSV file. If the table already exists, it will throw an error.
+  "Creates a table from a CSV file. If the table already exists, it will throw an error.
```

Comment:
maybe rename the function to `create-from-csv!` ?

Reply:
Yip! Already done in master https://github.com/metabase/metabase/pull/40367

Reply:
this diff looks like this branch will update master when merged